File: macos/QMK Toolbox/USB/Bootloader/BootloaderType.swift
import Foundation

@objc
public enum BootloaderType: Int {
    case apm32Dfu = 0
    case atmelDfu = 1
    case atmelSamBa = 2
    case avrIsp = 3
    case bootloadHid = 4
    case caterina = 5
    case gd32vDfu = 6
    case halfKay = 7
    case kiibohdDfu = 8
    case lufaHid = 9
    case lufaMs = 10
    case qmkDfu = 11
    case qmkHid = 12
    case stm32Dfu = 13
    case stm32duino = 14
    case usbAsp = 15
    case usbTinyIsp = 16
    case wb32Dfu = 17
    case none = 18
}

File: linux/src/qmk_toolbox/bootloader/bootloader_type.py
from enum import Enum, auto


class BootloaderType(Enum):
    APM32_DFU = auto()
    ATMEL_DFU = auto()
    ATMEL_SAM_BA = auto()
    BOOTLOAD_HID = auto()
    CATERINA = auto()
    GD32V_DFU = auto()
    HALFKAY = auto()
    KIIBOHD_DFU = auto()
    LUFA_HID = auto()
    LUFA_MS = auto()
    STM32_DFU = auto()
    STM32DUINO = auto()
    USBASP = auto()
    USBTINY_ISP = auto()
    AVR_ISP = auto()
    WB32_DFU = auto()
